src/gui.c

```diff
@@ -13841,6 +13911,7 @@ static void about_dialog(GtkMenuItem *menu_item, gpointer data)
 "This program is written by Keldon Jones, and the source code is licensed \
 under the GNU General Public License.\n\n\
 The interface enhancement release " RELEASE " is written by B. Nordli.\n\n\
+The Xeno Invasion expansion has been added by J.-R. Reinhard.\n\n\
 Race for the Galaxy was designed by Tom Lehmann and published by Rio Grande \
 Games.  All card and other art is copyrighted by Rio Grande Games.\n\n\
 Send bug reports to keldon@keldon.net");
```

Comment:
As Keldon is unresponsible at the moment, I suggest changing this line to "Report bugs to https://github.com/bnordli/rftg/issues."